spec/fixtures/files/xml/core/collectionobject1.xml

```diff
@@ -306,7 +306,7 @@
       <assocConceptGroup>
         <assocConcept>urn:cspace:core.collectionspace.org:conceptauthorities:name(concept):item:name(Test1599650854716)'Test'</assocConcept>
         <assocConceptNote>men</assocConceptNote>
-        <assocConceptType>Great </assocConceptType>
```

Comment:
Mapper strips preceding/trailing spaces. Made #43 to remind me to document this, in case any one expects to be able to import data with meaningful preceding/trailing spaces.